### DISCUSSION TITLE
有关路由权限，el-menu 的显示。

### DISCUSSION THREAD

**Discussion by llds66** [un-pany/v3-admin-vite#229] [Q&A] [answered]
已经设置好根据roles 控制路由页面权限，已经生效。但是el-menu没有完全隐藏。请问这是怎么回事呢？
![image](https://github.com/user-attachments/assets/b2b7d94b-440a-49de-bf57-20449da4531c)
![image](https://github.com/user-attachments/assets/12932e05-7a64-4b93-a62c-7afb822e83de)

**Comment by pany-ang** [comment, marked as answer]
给一个复现代码

**Reply by llds66** [reply]
我的父路由忘记给meta了
